pytest_timeout.py

```diff
@@ -329,14 +364,16 @@ def _parse_marker(marker):
     """
     if not marker.args and not marker.kwargs:
         raise TypeError("Timeout marker must have at least one argument")
-    timeout = method = func_only = NOTSET = object()
+    timeout = method = func_only = disable_debugger_detection = NOTSET = object()
```

Comment:
is there really any value in allowing this in the marker?  It would be simpler if that wasn't possible.

Reply:
I was attempting to match the pattern for other options. Thinking more about it now, I'm not seeing a need for it to be a marker parameter either.